rxandroidble/src/main/java/com/polidea/rxandroidble2/RxBleConnection.java

```diff
@@ -584,17 +584,28 @@ Completable requestConnectionPriority(
      */
     int getMtu();
 
+    /**
+     * Performs GATT read PHY operation.
+     *
+     * @return Observable emitting the read Tx and Rx values.
+     */
     @RequiresApi(26 /* Build.VERSION_CODES.O */)
     Single<PhyPair> readPhy();
 
     /**
      * Performs set preferred PHY request.
      *
+     * @param txPhy Sets the preferred transmitter (Tx) PHY.
+     * @param rxPhy Sets the preferred receiver (Rx) PHY.
+     * @param phyOptions Sets the preferred coding to use when transmitting on the LE Coded PHY.
      * @return Observable emitting whether the GATT operation succeeded or not.
```

Comment:
This is no longer the case. Maybe something like:
```suggestion
     * @return Observable emitting negotiated PHY values pair.
```
Feel free to change, English is not my first language :)

Reply:
Your English is better than most people's I know :)